test/others/criu-ns/run.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+
+import os
+import pty
+import shutil
+import subprocess
+import sys
+import time
+
+
+CRIU_BIN = "../../../criu/criu"
+CRIU_NS = "../../../scripts/criu-ns"
+
+
+def check_dumpdir():
+    if os.path.isdir("dumpdir"):
+        shutil.rmtree("dumpdir")
+    os.mkdir("dumpdir", 0o755)
+
+
+def create_pty():
+    fd_m, fd_s = pty.openpty()
+    return (os.fdopen(fd_m, "wb"), os.fdopen(fd_s, "wb"))
+
+
+def test_dump_and_restore_with_shell_job():
+    check_dumpdir()
+
+    open("running", "w").close()
+    create_pty()
```

Comment:
we don't use returned fds, do we need this call?

Reply:
Yeah, initially I removed the returned fd assignment to avoid the `CodeQL` warning. Now I can confirm the CI tests and the local tests are working fine without the returned fds. So, I'm removing this call.